applications/vane-network.md

```diff
@@ -0,0 +1,317 @@
+  The POC implemented most of things in customized existing pallets but they seemed too rigid and did not fit vane implementation vision, so that is why there is vane custom pallet asset. And other pallets are totally unique.
+
+  Note that we removed our initial first milestone, we will implement that without any cost. We strongly believe after the funding of Vane from web3 foundation we will be rady to go the market and prove our case by action that users need safety in their day to day crypto interactions ( transactions ) and blockchain can be used in social-commerce.
+
+  We have already deployed into Tanssi DanceBox testnet.
+
+
+  ### Note:
+  As per requirement of web3 grant deliverables is to submit an article. But as we do believe the purpose of the article is to communicate with users/community and we do think a video might communicate much better than an article, hence we will be providing a video containing a workshop, user experience usage and our experience in the web3 foundation grant. We do believe videos get better engagements and if we want to be heard, its better we do that.
+
+
+
+
+### Milestone 1 — Trasanction Delivery Tracking System with XCM & Pallet Tansaction Claim.
+ 
+
+- **Estimated duration:** 0.7 month
+- **FTE:**  2.5
+- **Costs:** 9,000 USD
+
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can (for example) spin up one of our Substrate nodes and send test transactions, which will show how the new functionality works. |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Video tutorial | We will publish an workshop on X/Twitter that explains [...] (what was done/achieved as part of the grant and how the product should be used) |
+| 1. | Transaction Delivery Tracking System | As vane-risk free transfer system involves the following parts.<br> Relay_chain xcm reserveTransfer to Vane, confirmation of sender and receiver's address,sending xcm instructions to relay/assethub for transacting asset from vane para account to receiver.<br> The following are notable issues which may arrive; Failure to deliver and execute XCM message to vane, and relay/assethub.<br><br> A tracking functionality of where the transaction stuck is essential and will be implemented as a struct of an enum and stored in transaction tracking storage with key of multi_id.<br> The updating will be based on queries of effect of what the transaction is supposed to do using XCM.<br>|
```

Comment:
Is this a pallet? Pls specify directly in the doc.

Reply:
Yes it is a pallet